CSharpFunctionalExtensions/Result/Dto/ResultDtoJsonConverterOfT.cs

```diff
@@ -0,0 +1,65 @@
+﻿#nullable enable
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace CSharpFunctionalExtensions
+{
+    public class ResultDtoJsonConverterFactory : JsonConverterFactory
```

Comment:
I think it's better to name it `ResultJsonConverterFactory`.

No need to add any clarification since later we will add `Result<T, TError>` and `UnitResult<TError>` serialization support directory to this class

Reply:
Rename